src/informatica/azext_informatica/aaz/latest/informatica/data_management/organization/_create.py

```diff
@@ -0,0 +1,588 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "informatica data-management organization create",
+)
+class Create(AAZCommand):
+    """Create a InformaticaOrganizationResource
+    """
```

Comment:
Please add examples for commands in aaz-dev-tools before release, you can following this [doc](https://azure.github.io/aaz-dev-tools/pages/usage/workspace-editor/#command-examples)

Reply:
Added!